pkgs/applications/audio/guayadeque/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, pkg-config
+, wrapGAppsHook
+, libgpod
+, wxGTK31
```

Comment:
`wxGTK31` is orphaned in nixpkgs and may be dropped very soon. Could you patch it for `wxGTK32`?